nautilus_trader/adapters/oanda/data.pyx

```diff
@@ -152,10 +152,11 @@ cdef class OandaDataClient(LiveMarketDataClient):
         # Schedule subscribed instruments update
         self._load_instruments()
 
-        self._update_instruments_handle: asyncio.Handle = self._loop.call_later(
-            delay=_SECONDS_IN_HOUR,  # Every hour
-            callback=self._subscribed_instruments_update,
-        )
+        # TODO(cs): Investigate a better way of doing this (coro hangs)
```

Comment:
I wonder if this was the same issue I was running into when starting a live session. I could never track it down. With this disabled, does it mean we can't run it live until this bug is fixed?

Reply:
Commenting this out just means that instruments aren't automatically updated every hour. A strategy is still able to make a request for instrument(s) as required however, so this should be fine for the moment.

There's actually a couple of asyncio related issues to clean up. You've probably noticed the pending task on stop with the CCXT adapter, I have to devote some time to investigating this further. At one stage they applied a 'fix' and it was working, then a later version seemed to regress - it may be a Nautilus issue too though.